ios/browser/api/ads/brave_ads.mm

```diff
@@ -1817,28 +1812,28 @@ - (void)notifyPendingObservers {
 }
 
 - (void)notifyDidInitializeAds {
-  if ([self isServiceRunning]) {
+  if (adsClientNotifier != nil) {
```

Comment:
`adsClientNotifier` can be accessed before `ads` instance is created, so we no need to check that `ads` is not null.